/mnt/hwfile/huqinghao/miniconda3/bin/python
2024-05-06 01:58:34,728 - INFO - Note: detected 128 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
2024-05-06 01:58:34,728 - INFO - Note: NumExpr detected 128 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-05-06 01:58:34,728 - INFO - NumExpr defaulting to 8 threads.
2024-05-06 01:58:36,445	INFO scripts.py:1182 -- Did not find any active Ray processes.

==========
== CUDA ==
==========

CUDA Version 12.1.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

*************************
** DEPRECATION NOTICE! **
*************************
THIS IMAGE IS DEPRECATED and is scheduled for DELETION.
    https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/support-policy.md

/usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_name_or_path" has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
INFO 05-06 01:58:49 api_server.py:202] vLLM API server version 0.3.3
INFO 05-06 01:58:49 api_server.py:203] args: Namespace(host=None, port=8000, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, served_model_name=None, lora_modules=[], delta_modules=[Delta(name='delta-1', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-1'), Delta(name='delta-2', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-2'), Delta(name='delta-3', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-3'), Delta(name='delta-4', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-4'), Delta(name='delta-5', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-5'), Delta(name='delta-6', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-6'), Delta(name='delta-7', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-7'), Delta(name='delta-8', local_path='/vllm/.idea/models/70b/llama2-chat-70b.4b75s128g-bitblas-tp_8-8')], swap_modules=[], chat_template=None, response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], model='/vllm/.idea/full_models/llama-70b', tokenizer=None, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=False, download_dir=None, load_format='auto', dtype='auto', kv_cache_dtype='auto', max_model_len=None, worker_use_ray=False, pipeline_parallel_size=1, tensor_parallel_size=8, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=16, enable_prefix_caching=False, seed=0, swap_space=4, gpu_memory_utilization=0.85, max_num_batched_tokens=None, max_num_seqs=256, max_logprobs=5, disable_log_stats=False, quantization=None, enforce_eager=True, max_context_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, enable_lora=False, max_loras=1, max_lora_rank=64, lora_extra_vocab_size=256, lora_dtype='auto', max_cpu_loras=32, enable_delta=True, max_deltas=4, max_cpu_deltas=32, max_delta_bitwidth=4, device='auto', image_input_type=None, image_token_id=None, image_input_shape=None, image_feature_size=None, scheduler_delay_factor=0.0, enable_prefetch=True, engine_use_ray=False, disable_log_requests=True, max_log_len=None)
2024-05-06 01:58:52,420	INFO worker.py:1749 -- Started a local Ray instance.
INFO 05-06 01:58:54 llm_engine.py:87] Initializing an LLM engine (v0.3.3) with config: model='/vllm/.idea/full_models/llama-70b', tokenizer='/vllm/.idea/full_models/llama-70b', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, lora_config=None, delta_config=DeltaConfig(max_deltas=4, max_bitwidth=4, delta_dtype=torch.int8, max_cpu_deltas=32, delta_extra_vocab_size=0, pack_factor=Fraction(2, 1), kernel=<QuantKernel.TRITON: 'triton'>), max_seq_len=4096, download_dir=None, load_format=auto, tensor_parallel_size=8, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, seed=0)
INFO 05-06 01:58:55 llm_engine.py:237] ignored verifying delta_config
Using bitblas
WARNING 05-06 02:01:17 models.py:48] Using bitblas delta modules
INFO 05-06 02:01:18 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=117799)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f85d3dc0c10>
[36m(RayWorkerVllm pid=117799)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=117799)[0m     self.remove()
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=117799)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=117799)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118675)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f83029a0c10>
[36m(RayWorkerVllm pid=118675)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118675)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118675)[0m     self.remove()
[36m(RayWorkerVllm pid=118675)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118675)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118675)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118774)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f319d5bcc10>
[36m(RayWorkerVllm pid=118774)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118774)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118774)[0m     self.remove()
[36m(RayWorkerVllm pid=118774)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118774)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118774)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=117484)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7fb6ce110c10>
[36m(RayWorkerVllm pid=117484)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=117484)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=117484)[0m     self.remove()
[36m(RayWorkerVllm pid=117484)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=117484)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=117484)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118573)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f7f009c0c10>
[36m(RayWorkerVllm pid=118573)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118573)[0m     self.remove()
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118573)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118573)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118774)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f319d5bcc10>
[36m(RayWorkerVllm pid=118774)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118774)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118774)[0m     self.remove()
[36m(RayWorkerVllm pid=118774)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118774)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118774)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=117799)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f85d3dc0c10>
[36m(RayWorkerVllm pid=117799)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=117799)[0m     self.remove()
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=117799)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=117799)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118573)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f7f009c0c10>
[36m(RayWorkerVllm pid=118573)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118573)[0m     self.remove()
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118573)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118573)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=117484)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7fb6ce110c10>
[36m(RayWorkerVllm pid=117484)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=117484)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=117484)[0m     self.remove()
[36m(RayWorkerVllm pid=117484)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=117484)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=117484)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=117799)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f85d3dc0c10>
[36m(RayWorkerVllm pid=117799)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=117799)[0m     self.remove()
[36m(RayWorkerVllm pid=117799)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=117799)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=117799)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118573)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f7f009c0c10>
[36m(RayWorkerVllm pid=118573)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=118573)[0m     self.remove()
[36m(RayWorkerVllm pid=118573)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=118573)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=118573)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=118313)[0m Using bitblas
[36m(RayWorkerVllm pid=118675)[0m Using bitblas
[36m(RayWorkerVllm pid=118774)[0m Using bitblas
[36m(RayWorkerVllm pid=117484)[0m Using bitblas
[36m(RayWorkerVllm pid=117651)[0m Using bitblas
[36m(RayWorkerVllm pid=117799)[0m Using bitblas
[36m(RayWorkerVllm pid=118573)[0m Using bitblas
[36m(RayWorkerVllm pid=118675)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118675)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=118774)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118774)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=117484)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=117484)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=117799)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=117799)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=118573)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118573)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=117799)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=117799)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=118774)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118774)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=118573)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118573)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=117484)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=117484)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/ff63f81ac74960937196963ebe01df1febe08ea917b3fa029bd8b3ce3ee821f1/tvm_rt_mod'
[36m(RayWorkerVllm pid=117799)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=117799)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=118573)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=118573)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/0c3b297407291488ea177fe38613ff452d2665fb8ee52d05966f020178d526f2/tvm_rt_mod'
[36m(RayWorkerVllm pid=118313)[0m WARNING 05-06 02:03:13 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:03:13 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=117651)[0m WARNING 05-06 02:03:13 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:03:13 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=118675)[0m WARNING 05-06 02:03:19 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:03:19 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=118774)[0m WARNING 05-06 02:03:23 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:03:23 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=117484)[0m WARNING 05-06 02:03:25 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:03:25 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=117799)[0m WARNING 05-06 02:03:27 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:03:27 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=118573)[0m WARNING 05-06 02:03:32 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:03:32 selector.py:16] Using FlashAttention backend.
INFO 05-06 02:03:58 model_runner.py:155] Loading model weights took 16.0630 GB
INFO 05-06 02:03:58 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:04:02 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:04:02 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:04:02 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:04:02 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:04:02 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:04:02 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:04:03 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:04:03 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:04:03 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:04:03 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:04:03 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:04:03 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:04:03 model_runner.py:155] Loading model weights took 16.0630 GB
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:04:03 worker_manager.py:293] Starting prefetching thread
WARNING 05-06 02:04:08 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=118774)[0m WARNING 05-06 02:04:11 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=118313)[0m WARNING 05-06 02:04:12 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=117799)[0m WARNING 05-06 02:04:12 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=117484)[0m WARNING 05-06 02:04:12 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=117651)[0m WARNING 05-06 02:04:12 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=118573)[0m WARNING 05-06 02:04:13 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=118675)[0m WARNING 05-06 02:04:13 quant_linear_triton.py:557] We should warm up the kernel here 
INFO 05-06 02:04:31 ray_gpu_executor.py:266] # GPU blocks: 48922, # CPU blocks: 6553
INFO 05-06 02:04:36 block_manager.py:264] disable automatic prefix caching
WARNING 05-06 02:04:37 serving_chat.py:373] No chat template provided. Chat API will not work.
INFO:     Started server process [109653]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     ::1:50028 - "GET /sysinfo HTTP/1.1" 200 OK
INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=117484)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=117651)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=117799)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=118313)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.4.safetensors']
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=118573)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.5.safetensors']
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=118675)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.6.safetensors']
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:04:43 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=118774)[0m WARNING 05-06 02:04:43 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:04:43 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.7.safetensors']
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:04:43 models.py:268] [main] Lossless Compression Disabled
INFO 05-06 02:04:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:04:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:05:03 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 19.755 seconds
INFO 05-06 02:05:03 worker_manager.py:353] Main thread loading delta 8 done
INFO 05-06 02:05:03 worker_manager.py:368] [1714932303.8606312] activating delta
INFO 05-06 02:05:04 worker_manager.py:373] [1714932304.1108167] CPU -> GPU time: 0.2501
INFO 05-06 02:05:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:05:35 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 51.910 seconds
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:05:35 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:05:35 worker_manager.py:368] [1714932335.9864814] activating delta
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:05:36 worker_manager.py:373] [1714932336.2415667] CPU -> GPU time: 0.2550
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:05:48 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 64.198 seconds
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:05:48 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:05:48 worker_manager.py:368] [1714932348.2645154] activating delta
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:05:48 worker_manager.py:373] [1714932348.5294626] CPU -> GPU time: 0.2649
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:05:49 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 65.717 seconds
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:05:49 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:05:49 worker_manager.py:368] [1714932349.783293] activating delta
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:05:50 worker_manager.py:373] [1714932350.0405443] CPU -> GPU time: 0.2572
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:05:50 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 66.307 seconds
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:05:50 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:05:50 worker_manager.py:368] [1714932350.3709905] activating delta
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:05:50 worker_manager.py:373] [1714932350.632158] CPU -> GPU time: 0.2611
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:05:56 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 72.563 seconds
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:05:56 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 72.547 seconds
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:05:56 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:05:56 worker_manager.py:368] [1714932356.642729] activating delta
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:05:56 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:05:56 worker_manager.py:368] [1714932356.6687808] activating delta
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:05:56 worker_manager.py:373] [1714932356.9371743] CPU -> GPU time: 0.2683
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:05:56 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 73.052 seconds
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:05:56 worker_manager.py:373] [1714932356.902226] CPU -> GPU time: 0.2594
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:05:57 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:05:57 worker_manager.py:368] [1714932357.1113439] activating delta
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:05:57 worker_manager.py:373] [1714932357.3683817] CPU -> GPU time: 0.2570
INFO 05-06 02:06:02 metrics.py:276] Avg prompt throughput: 4.2 tokens/s, Avg generation throughput: 1.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:13 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:18 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:06:55 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:07:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 2.3 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO:     ::1:50030 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=117484)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=117651)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=117799)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=118313)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.4.safetensors']
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=118573)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.5.safetensors']
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=118675)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.6.safetensors']
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:07:03 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=118774)[0m WARNING 05-06 02:07:03 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:07:03 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.7.safetensors']
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:07:03 models.py:268] [main] Lossless Compression Disabled
INFO 05-06 02:07:15 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 12.731 seconds
INFO 05-06 02:07:16 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:09:53 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 170.463 seconds
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:09:53 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 170.524 seconds
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:09:53 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:09:53 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:09:55 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 172.223 seconds
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:09:55 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:00 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 177.862 seconds
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:01 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:04 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 181.664 seconds
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:05 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:08 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 185.449 seconds
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:08 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 185.446 seconds
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:08 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:08 worker_manager.py:353] Main thread loading delta 6 done
INFO 05-06 02:10:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=117484)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=117651)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=117799)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=118313)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=118573)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=118675)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:10:11 worker_manager.py:325] Adding delta 1 to prefetching queue: [4, 1]
[36m(RayWorkerVllm pid=118774)[0m WARNING 05-06 02:10:11 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=117484)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=117651)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=117799)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.4.safetensors']
[36m(RayWorkerVllm pid=118313)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.5.safetensors']
[36m(RayWorkerVllm pid=118573)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.6.safetensors']
[36m(RayWorkerVllm pid=118675)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:10:11 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.7.safetensors']
[36m(RayWorkerVllm pid=118774)[0m INFO 05-06 02:10:11 models.py:268] [main] Lossless Compression Disabled
INFO 05-06 02:17:32 models.py:324] Disk -> CPU: Loaded 4223.53 MiB in 440.674 seconds
INFO 05-06 02:17:32 worker_manager.py:353] Main thread loading delta 1 done
INFO 05-06 02:17:32 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-06 02:17:32 models.py:268] [prefetching] Lossless Compression Disabled
WARNING 05-06 02:17:38 worker_manager.py:335] Missed prefetch, fall back to loading delta 4 now
WARNING 05-06 02:17:38 worker_manager.py:344] Discarding prefetching delta 4
INFO 05-06 02:17:38 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-06 02:17:38 models.py:268] [main] Lossless Compression Disabled
slurmstepd: error: *** JOB 3666634 ON HOST-10-140-60-3 CANCELLED AT 2024-05-06T02:18:53 ***
